File: C#/ex3.cs
using System;

class Program
{
    static void Main()
    {
        int N = Convert.ToInt32(Console.ReadLine());
        int sum = 0;
        int product = 1;
        while (N > 0)
        {
            sum += N % 10;
            product *= N % 10;
            N /= 10;
        }
        Console.WriteLine($"{sum} {product} {Math.Abs(sum - product)}");
    }
}

File: Go/ex3.go
package main

import "fmt"

func main() {
    var N int
    fmt.Scan(&N)

    var sum int = 0
    var product int = 1

    for N > 0 {
        sum += N % 10
        product *= N % 10
        N /= 10
    }
    fmt.Printf("%d %d %d\n", sum, product, abs(sum-product))
}

func abs(x int) int {
    if x < 0 {
        return -x
    }
    return x
}